scaleway/table_scaleway_baremetal_server.go

```diff
@@ -0,0 +1,279 @@
+		GetMatrixItemFunc: BuildZoneList,
+		List: &plugin.ListConfig{
+			Hydrate: listBaremetalServers,
+			KeyColumns: []*plugin.KeyColumn{
+				{
+					Name:    "name",
+					Require: plugin.Optional,
+				},
+				{
+					Name:    "zone",
+					Require: plugin.Optional,
+				},
+			},
+		},
+		Get: &plugin.GetConfig{
+			Hydrate:    getBaremetalServer,
+			KeyColumns: plugin.AllColumns([]string{"id", "zone"}),
+		},
+		Columns: []*plugin.Column{
+			{
+				Name:        "name",
+				Description: "The user-defined name of the server.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "id",
+				Description: "A unique identifier of the server.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("ID"),
+			},
+			{
+				Name:        "description",
+				Description: "The description of the server.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "status",
+				Description: "The current state of the server.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Status").Transform(transform.ToString),
+			},
+			{
+				Name:        "updated_at",
+				Description: "The time when the server was updated.",
+				Type:        proto.ColumnType_TIMESTAMP,
+			},
+			{
+				Name:        "created_at",
+				Description: "The time when the server was created.",
+				Type:        proto.ColumnType_TIMESTAMP,
+			},
+			{
+				Name:        "boot_type",
+				Description: "The server boot type.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("BootType").Transform(transform.ToString),
+			},
+			{
+				Name:        "offer_name",
+				Description: "The offer name of the server.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "offer_id",
+				Description: "The offer ID of the server.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "tags",
+				Description: "A list of tags associated with the server.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "ips",
+				Description: "A list of IPs associated with the server.",
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("IPs"),
+			},
+			{
+				Name:        "domain",
+				Description: "The domain of the server.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "install",
+				Description: "The install of the server.",
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("Install"),
+			},
+			{
+				Name:        "ping_status",
+				Description: "The status of the ping.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("PingStatus").Transform(transform.ToString),
+			},
+			{
+				Name:        "options",
+				Description: "The options of the server.",
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("Options"),
```

Comment:
Do we need this transform, since the default transform for the plugin is `transform.FromCamel()` (which should be able to convert `Options` to `options` I believe)?